### DISCUSSION TITLE
Split display (dual page pendant)

### DISCUSSION THREAD

**Discussion by mlarchet** [gotson/komga#1565] [Ideas]
### Describe your suggested feature

I have some scans that are full width of the original album : one picture with two pages side by side.
Is there any way to display these images in 'split mode', the exact opposite of double page ?

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
No. It's been requested in the past #509 #40
